.nvmrc

```diff
@@ -1 +1 @@
-16.13.0
+18.18.2
```

Comment:
We should probably go all the way to `18.20.2` while we're at it (https://nodejs.org/en/about/previous-releases)